### DISCUSSION TITLE
The category "Polls" is not available in the Discussions section

### DISCUSSION THREAD

**Discussion by elisbyberi** [modular/modular#99] [General]
We should consider creating a poll within the community to gather feedback on potential new features. This would allow us to gather multiple opinions on each proposed feature.

**Comment by lattner** [comment]
Community engagement is super important, but this isn't how we're prioritizing features.  We're prioritizing features in terms of the dependence graph we need to build out to get a technically great product.  Please check out our roadmap for the big bricks we need to get in place.

**Reply by TZDak** [reply]
It would probably still be helpful to the community to have a place for polling each other with regard to various Mojo-related topics or opinions. Keep in mind that for the most part we're here to help and, at least in some cases, we can do so more efficiently if we work together than if we just talk at the developers and ignore each other. So if it wouldn't be difficult to add (which I suspect it would not), I would also recommend adding it. Let us help more efficiently and you'll be glad you did. You're not alone in this.
